api/cont.js

```diff
@@ -114,7 +114,17 @@
                 continuation
                     .extend({source: sourceFragment}, idx, sourceLength, value)
                     .evalReference();
-            });
+            };
+
+            if (typeof source.length === 'number') {
+                $.each(source, eachCallback);
+            }
+            else {
```

Comment:
The reason why I tried not using `$.each` is due to the following reasons:
1. `$.each` is working on the previous template object
2. `$.each` is defined before the iteration
3. we never reach the `$.each` callback function

It would've make sense had `$.each` was failing to iterate every time, however, it was failing only sometimes. @benlast and I were thinking there may be some race condition here that made `$.each` failed sometimes. So we took out `$.each` just to see if we can reproduce the same error.

The `source.length` logic is grabbed from zepto's definition of `likeArray` function

Zepto's `$.each`: https://github.com/mobify/zepto/blob/master/src/zepto.js#L232
Zepto's `likeArray`: https://github.com/mobify/zepto/blob/master/src/zepto.js#L55